docker-compose/Makefile

```diff
@@ -163,7 +173,11 @@ destroy: stop rm
 	docker ps | grep whisk | awk '{print $$1}' | xargs docker stop | xargs docker rm
 	echo "cleaning dangling docker volumes ... "
 	docker volume ls -qf dangling=true | xargs docker volume rm
-	rm -rf ~/tmp/openwhisk
+	if [ "$(UNAME_STR)" = "Linux" ]; then \
+	  sudo rm -rf ~/tmp/openwhisk ;\
```

Comment:
is the `sudo` here for Travis ? Else I'm reading that the current use can't delete things in its own home folder ?

Reply:
They can sure but in this case docker is writing data there. Docker is running as root on linux, so when it writes the data files for couch here, the user can't actually delete them. Detecting the current user (uid), getting it created in then injecting that into the running container is a total pain.

Reply:
I'm happy to change the Couch stuff to use a docker volume which will relieve that issue but I didn't want to make such a "big" change.

Reply:
thanks for explaining. I also prefer the `sudo` approach in this case.